shared/go/pkg/valueobject/shortcode.go

```diff
@@ -16,15 +16,18 @@
 
 package valueobject
 
-import "fmt"
+import (
+	"fmt"
+	"strings"
+)
 
 type ShortCode struct {
 	value string
 }
 
 // NewShortCode creates a new valid short code object.
 func NewShortCode(value string) (ShortCode, error) {
```

Comment:
Should strictly test for two hexadecimal digits

Reply:
done in a1634d0